CONTRIBUTORS.md

```diff
@@ -2944,12 +2944,6 @@
 
 - [@shubhamgoel90](https://github.com/shubhamgoel90)
 
-<<<<<<< HEAD
-
-=======
-
->>>>>>> 730949ce3e211ac66e330835d16a1001d87c7e09
-
```

Comment:
done

Reply:
Yes, but what about the other files? For example, `README.md` (see my requested changes/questions before)?
Also, please update your fork to the upstream repository, see https://github.com/zero-to-mastery/start-here-guidelines#a-guide-to-get-started Step 4.